### PR TITLE
Update release notes for v0.6.0 tag

### PR DESCRIPTION
**Pull request by kbeker**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+- Feature add signup email confirmation ([#308](https://github.com/Code-Poets/sheetstorm/pull/308))
+- Feature better export of reports ([#331](https://github.com/Code-Poets/sheetstorm/pull/331))
+- Feature mitigating DDoS attack with nginx ([#297](https://github.com/Code-Poets/sheetstorm/pull/297))
+- Feature rebuild register interface ([#283](https://github.com/Code-Poets/sheetstorm/pull/283))
 
 ##### Bugfixes
-- Update `Django` to `2.2.2` to mitigate a security issue
+
+- Bugfix add a missing file that allows you to reset database in the virtual machine ([#338](https://github.com/Code-Poets/sheetstorm/pull/338))
+- Bugfix add security for exporting own reports ([#340](https://github.com/Code-Poets/sheetstorm/pull/340))
```
Comment (dybi) [inline RELEASE-NOTES.md]: it is not needed - feature wasn't present in previous release

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+- Feature add possibility to export monthly reports for every user ([#332](https://github.com/Code-Poets/sheetstorm/pull/332))
+- Feature add signup email confirmation ([#308](https://github.com/Code-Poets/sheetstorm/pull/308))
+- Feature better export of reports ([#331](https://github.com/Code-Poets/sheetstorm/pull/331))
+- Feature mitigating DDoS attack with nginx ([#297](https://github.com/Code-Poets/sheetstorm/pull/297))
+- Feature rebuild register interface ([#283](https://github.com/Code-Poets/sheetstorm/pull/283))
 
 ##### Bugfixes
-- Update `Django` to `2.2.2` to mitigate a security issue
+
+- Bugfix add a missing file that allows you to reset database in the virtual machine ([#338](https://github.com/Code-Poets/sheetstorm/pull/338))
```
Comment (dybi) [inline RELEASE-NOTES.md]: not sure about this one, please see comment below

File: RELEASE-NOTES.md (hunk 1 above)
Comment (kbeker) [reply RELEASE-NOTES.md]: You're right :) Deleted

File: RELEASE-NOTES.md (hunk 2 above)
Comment (kbeker) [reply RELEASE-NOTES.md]: Why? Virtual machine was added in last release and reseting database was missing in previous configuration

Comment (dybi) [reply RELEASE-NOTES.md]: OK

**Pull request merged**